makelib/golang.mk

```diff
@@ -198,3 +198,6 @@ $(GOLANGCILINT):
 	@mv $(TOOLS_HOST_DIR)/tmp-golangci-lint/golangci-lint $(GOLANGCILINT) || $(FAIL)
 	@rm -fr $(TOOLS_HOST_DIR)/tmp-golangci-lint
 	@$(OK) installing golangci-lint-v$(GOLANGCILINT_VERSION) $(SAFEHOSTPLATFORM)
+
+vendor: modules.download
+vendor.check: modules.check
```

Comment:
Could you add a comment explaining that these are deprecated and folks should prefer the `modules` variants.